bloc_library/pubspec.yaml

```diff
@@ -7,13 +7,21 @@ environment:
 dependencies:
   meta: ">=1.1.0 <2.0.0"
   equatable: ^0.2.0
-  flutter_bloc: ^0.7.0
+  flutter_bloc: ^0.10.0
   flutter:
     sdk: flutter
+
+dependency_overrides:
```

Comment:
flutter_bloc ^0.10.0 depends on rxdart ^0.21.0 so this change was made to allow the bloc_library to work without forcing an upgrade in other examples to user rxdart ^0.21.0.